tests/core/test_geometric_objects.py

```diff
@@ -429,6 +429,12 @@ def test_tube():
         pv.Tube(pointa, (10, 1.0))
 
 
+@pytest.mark.parametrize('capping', [True, False])
+def test_tube_capping(capping: bool):
+    tube: pv.PolyData = pv.Tube(capping=capping).clean().triangulate()
```

Comment:
Had to clean due to duplicated points at the cylinder end borders

Reply:
```suggestion
    # Had to clean due to duplicated points at the cylinder end borders.
    tube: pv.PolyData = pv.Tube(capping=capping).clean().triangulate()
```